packages/SwingSet/test/vat-admin/terminate/bootstrap-terminate.js

```diff
@@ -1,38 +1,46 @@
 import { E } from '@endo/eventual-send';
 import { Far } from '@endo/marshal';
 
-export function buildRootObject(vatPowers, vatParameters) {
+export function buildRootObject(vatPowers) {
   const { testLog } = vatPowers;
-  const [mode, critical, dynamic] = vatParameters.argv;
+  let vatAdminService;
+  let staticVats;
+  let criticalVatKey;
+  let root;
+  let adminNode;
 
   const self = Far('root', {
-    async bootstrap(vats, devices) {
+    bootstrap(vats, devices) {
+      vatAdminService = E(vats.vatAdmin).createVatAdminService(
```

Comment:
Huh, interseting, I didn't realize the original test wasn't `await`ing, so it holds onto `vatAdminServiceP` rather than the actual presence. Eh, it's different than our other tests, but probably good to have some diversity.